docs/en/stack/ml/df-analytics/dfa-classification.asciidoc

```diff
@@ -64,10 +70,33 @@ otherwise the {classanalysis} may not provide the best predictions. Read
 The ensemble algorithm that we use in the {stack} for {classification} is a type 
 of boosting called boosted tree regression model which combines multiple weak 
 models into a composite one. We use decision trees to learn to predict the 
-probability that a data point belongs to a certain class.
+probability that a data point belongs to a certain class. A sequence of decision 
+trees are trained and every decision tree learns from the mistakes of the 
+previous one. Every tree is an iteration of the last one, hence it improves the 
+decision made by the previous tree.
 //end::classification-algorithms[]
 
 
+[discrete]
+[[dfa-classification-performance]]
+==== {classification-cap} performance
+
+As a rule of thumb, a {classanalysis} with many classes takes more time to run 
```

Comment:
> As a rule of thumb...

This sentence seems like it should be at the start of the next paragraph, not this one. This paragraph seems to be talking mostly about the number of documents, whereas the next one is talking about the number of classes.